imagearrayapp/app/src/main/java/com/gbbeard/arrayimageapp/MainActivity.java

```diff
@@ -0,0 +1,125 @@
+package com.gbbeard.arrayimageapp;
```

Comment:
Nice job!  One bug: your `1/4` indicator is not incrementing/decrementing properly.  If I click through all four images and then click `next` again, it loops back to the first image very nicely, but it tells me I am looking at photo `5/4`.  The same thing happens if I loop using the `previous` button - I get a `-1/4`